### PR TITLE
CSS Module landing page: filter effects

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/css/filter_effects/index.md [hunk 1]
```diff
@@ -8,30 +8,59 @@ tags:
...
-  - css.properties.filter
+spec-urls: 
+  - https://drafts.fxtf.org/filter-effects-2
+  - https://drafts.fxtf.org/filter-effects-1
 ---
 
 {{CSSRef}}
 
-**Filter Effects** is a module of CSS that defines a way of processing an element's rendering before it is displayed in the document.
+The **CSS filter effects** module defines a way of processing an element's rendering before it is displayed in the document.
```
Comment (dipikabh) [inline files/en-us/web/css/filter_effects/index.md:18]:
```suggestion
The properties in the **CSS filter effects** module let you define a way of processing an element's rendering before the element is displayed in the document. Examples of such effects include blurring and changing the intensity of the color of an element.
```

File: files/en-us/web/css/filter_effects/index.md [hunk 2]
```diff
@@ -8,30 +8,59 @@ tags:
...
 
-## Reference
+### Filter effects in action
+
+Play with the various sliders to apply filter effects to the image below:
```
Comment (dipikabh) [inline files/en-us/web/css/filter_effects/index.md:22]: I am not seeing any changes in the image when I move the sliders (chrome, firefox)

File: files/en-us/web/css/filter_effects/index.md [hunk 3]
```diff
@@ -8,30 +8,59 @@ tags:
...
+- {{cssxref("&lt;filter-function&gt;")}} data type
+
+- {{cssxref("background-image")}} CSS property
+- {{cssxref("background-blend-mode")}} CSS property
+- {{cssxref("mix-blend-mode")}} CSS property
+
+- {{glossary("interpolation")}} glossary term
 
-- {{cssxref("&lt;filter-function&gt;")}}
+- SVG {{cssxref("color-interpolation-filters")}} property
```
Comment (dipikabh) [inline files/en-us/web/css/filter_effects/index.md:57]: how about 

```suggestion
- {{cssxref("color-interpolation-filters")}} SVG property
```

File: files/en-us/web/css/filter_effects/index.md [hunk 4]
```diff
@@ -8,30 +8,59 @@ tags:
   - Guide
   - Overview
   - Reference
-browser-compat:
-  - css.properties.backdrop-filter
-  - css.properties.filter
+spec-urls: 
+  - https://drafts.fxtf.org/filter-effects-2
+  - https://drafts.fxtf.org/filter-effects-1
```
Comment (dipikabh) [inline files/en-us/web/css/filter_effects/index.md:13]: these links are showing as "Unknown specification" in the table

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @estelle! I don't see the sample in action and a few more comments

File: files/en-us/web/css/filter_effects/index.md (hunk 4 above)
Comment (teoli2003) [reply files/en-us/web/css/filter_effects/index.md:13]: There should be only one, and I think it should be: https://drafts.fxtf.org/filter-effects/

Comment (estelle) [inline files/en-us/web/css/filter_effects/index.md:13]:
```suggestion
  - https://drafts.fxtf.org/filter-effects
```

File: files/en-us/web/css/filter_effects/index.md (hunk 2 above)
Comment (estelle) [reply files/en-us/web/css/filter_effects/index.md:21]: https://github.com/mdn/css-examples/pull/116 fixes it. fingers crossed

File: files/en-us/web/css/filter_effects/index.md [hunk 5]
```diff
@@ -8,30 +8,58 @@ tags:
   - Guide
   - Overview
   - Reference
-browser-compat:
-  - css.properties.backdrop-filter
-  - css.properties.filter
+spec-urls: 
+  - https://drafts.fxtf.org/filter-effects
```
Comment (estelle) [inline files/en-us/web/css/filter_effects/index.md:12]:
```suggestion
  - https://drafts.fxtf.org/filter-effects-2/
```

File: files/en-us/web/css/filter_effects/index.md (hunk 4 above)
Comment (estelle) [reply files/en-us/web/css/filter_effects/index.md]: https://drafts.fxtf.org/filter-effects/ doesn't have backdrop-filter, neither does -1. And -2 doesn't have filter functions, so we need both here.

**Review by dipikabh — APPROVED** [review]

**Comment by dipikabh** [comment]
- PR test is failing here:
    > error: FixableError: files/en-us/web/css/filter_effects/george_floyd_memorial_sm.jpg can be removed, because it is not mentioned in files/en-us/web/css/filter_effects/index.md

- The updates look good to me, so approved. I guess the changes in css-examples will take a while to show up.

- Not a blocker for this pull request, but in general, I am not feeling so sure about the heading “Related concepts” any more. The list of properties, data types, functions in the "Related concepts" section are “Related” to the current module page but they're not really “concepts”. A concept topic would be something guide’y. The list in this section, on the other hand, are just links to related reference pages.
Would it be better to call the heading “Related references”?

**Pull request merged**